feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go

```diff
@@ -400,9 +392,9 @@ func TestBurstyTraffic(t *testing.T) {
 				counters["dutQosOctetsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).TransmitOctets().State())
 				counters["dutQosDroppedPktsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedPkts().State())
 				//counters["dutQosDroppedOctetsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedOctets().State())
-				if !deviations.QOSDroppedOctets(dut) {
```

Comment:
This deviation is currently used by other vendors too, so we can remove them yet.

Reply:
This is in cisco specific config code, where we are not using the deviations, as we support Dropped Octets now
So this should not affect other vendors.

Reply:
Added back the deviations

Reply:
Also added the OTG tests